.buildkite/macos.rayci.yml

```diff
@@ -23,6 +23,32 @@ steps:
     commands:
       - ./ci/ray_ci/macos/macos_ci_build.sh build_aarch64
 
+  - label: ":ray: core: :mac: small & large flaky tests"
```

Comment:
looks great overall!

one last thing, could you move these flaky test blocks to the very bottom of this file (rather than at the top)?

Reply:
done!